src/style.css

```diff
@@ -2,2 +1,70 @@
+.ttq {
+  margin: 0;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  height: 60px;
+  position: relative;
+}
+
+.nav_item {
+  display: inline-block;
+  margin-left: 20px;
+}
+
+.nav_item:first-child {
+  margin-left: 0;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  top: 10px;
+}
+
+.nav_link {
+  text-decoration: none;
+  color: black;
+  position: relative;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  text-transform: uppercase;
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
You've specified a font-family for your '.nav_link' class but you haven't provided a fallback font. It's a good practice to specify a generic family as a fallback in case the primary font is not available. For example, you can modify this line to: `font-family: Roboto, Arial, sans-serif, sans-serif;`